loctool.js

```diff
@@ -142,7 +144,8 @@ var settings = {
     xliffsDir: ".",
     xliffVersion: 1.2,
     localizeOnly: false,
-    projectType: "web"
+    projectType: "web",
+    exclude: ["node_modules", ".git", ".svn"]
```

Comment:
Should probably be

`exclude: ["**/node_modules", "**/.git", "**/.svn"]`

so that we ignore them at all levels